src/test/java/net/snowflake/ingest/streaming/internal/SnowflakeStreamingIngestClientTest.java

```diff
@@ -950,28 +950,30 @@ public void testRegisterBlobChunkLimit() throws Exception {
         1, client.partitionBlobListForRegistrationRequest(createTestBlobMetadata(3, 95, 2)).size());
     assertEquals(
         2,
-        client.partitionBlobListForRegistrationRequest(createTestBlobMetadata(3, 95, 2, 1)).size());
+        client
```

Comment:
The changes in this test can be reverted?

Reply:
Yes, reverted.